structure/0048-outreach-team.md

```diff
@@ -0,0 +1,71 @@
+# Outreach Team
+
+This document has been made according to the process outlined in [RFC
+0006][RFC6].
+
+## Summary
+
+The Outreach Team is a subteam of the Community Team in charge of outreach
+and communications within and outside the Quilt Project. This team is also
+responsible for working with the development teams to communicate the
+current status of development progress with the community and with any
+partners. In essence, it may be thought of as a "public relations
+department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, announcements, or team to-do lists.
+Historically, this has been managed by the Community Team as a whole.
+However, communications don't fall within the realm of the current Quilt
+community team, and thus this should have its own separate subteam.
+
+## Explanation
+
+The Outreach Team will be responsible for all communications outside of and
+within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts and announcements
+- managing the official QuiltMC social medias, with the exception of Discord
+- managing relations with other platforms and communities outside of a
+  moderation context
+- writing and publishing general status updates on the progress and
+  development of Quilt
+- assisting in management of public development boards and to-do lists,
+  including GitHub project boards
+- managing the content of the website, not including the functionality
```

Comment:
Why not? Who would be responsible for this if not the outreach team?

Reply:
Sorry for taking ages to respond to this - this is Community Tooling, no?